lars-reimann/api-editor
# refactor!: cleanup package-parser structure

### Summary of Changes

* Collect all model files in one location
* Properly "encapsulate" model declarations
* Replace uses of the `UsageStore` with the `UsageCountStore` and remove the former
* Remove the `improve` command (superseded by the `annotations` command.

### Testing Instructions

Check the created files:
```
cd package-parser
poetry run parse-package all -p package_parser -s package_parser -c package_parser -o out
```
